### ISSUE TITLE
[Question]  first stage checkpoint for llava-v1.5-13b

### ISSUE THREAD

**Issue by Maxlinn** [haotian-liu/LLaVA#542]
### Question

hi liu and the team, thanks for your hard and the team, llava is amazing!

recently llava-v1.5-13b has been released and its vision encoder resolution has been increased to 336px, which is `openai/clip-vit-large-patch14-336`([clue](https://huggingface.co/liuhaotian/llava-v1.5-13b/blob/main/config.json#L24)).

i want to train a model like llava-v1.5-13b, but due to my limit resources, i want to continue train from your llava-v1.5-13b's first-stage checkpoint(i.e. trained on `558K filtered image-text pairs from LAION/CC/SBU,`, but not trained on instructions), but it seems does not released.

i discovered `liuhaotian/llava-pretrain-llama-2-13b-chat` under your hf account, but it is adapter only and its resolution is still 224px, which is `openai/clip-vit-large-patch14`([clue](https://huggingface.co/liuhaotian/llava-pretrain-llama-2-13b-chat/blob/main/config.json#L19)). due to the difference of vision model, it can not be what i want.

to sum up, do you plan to release the first-stage checkpoint of `llava-v1.5-13b`(trained on caption, but not tuned on instructions)? that would much accelerates my research, much thanks!

**Comment by haotian-liu** [comment]
Hi, not sure if you are asking for the projector weights of the first-stage? It is released here. Or do you mean a version that uses 224x224 image resolution?

![shot](https://github.com/haotian-liu/LLaVA/assets/6631389/f88bf9e4-0bf6-4f3f-82bb-728cc5ae1599)

**Comment by Maxlinn** [comment]
thanks, i must have missed it while reading, deeply sorry for the inconvenience!
